samples/msal-browser-samples/VanillaJSTestApp2.0/app/IE11/authConfig.js

```diff
@@ -0,0 +1,21 @@
+// Config object to be passed to Msal on creation
+const msalConfig = {
+    auth: {
+        clientId: "3fba556e-5d4a-48e3-8e1a-fd57c12cb82e",
+        authority: "https://login.windows-ppe.net/common/"
+    },
+    cache: {
+        cacheLocation: "localStorage", // This configures where your cache will be stored
+        storeAuthStateInCookie: true, // Set this to "true" if you are having issues on IE11 or Edge
```

Comment:
This would be set to the value of `isIE || isEdge` (or remove those variables)

Reply:
Done